.claude/hooks/pre-git-add.sh

```
@@ -0,0 +1,25 @@
+#!/bin/sh
+# Pre-git-add hook for Claude Code - runs bun check before git add
+
+read json
+tool_name=$(echo "$json" | jq -r '.tool_name // empty' 2>/dev/null)
+command=$(echo "$json" | jq -r '.tool_input.command // empty' 2>/dev/null)
+
+# Check if this is a git add command
+if [ "$tool_name" = "Bash" ] && echo "$command" | grep -q "^git add"; then
+  cd /home/ubuntu/Projects/certquiz
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)